app/controllers/case_court_reports_controller.rb

```diff
@@ -59,12 +58,20 @@ def set_casa_case
     @casa_case = CasaCase.find_by(case_number: params[:id])
   end
 
+  def assigned_cases
+    @assigned_cases = if current_user.volunteer?
+                        CasaCase.actively_assigned_to(current_user)
+                      else
+                        current_user.casa_org.casa_cases.active
+                      end
+  end
+
   def generate_report_to_string(casa_case)
     return unless casa_case
 
     casa_case.casa_org.open_org_court_report_template do |template_docx_file|
       court_report = CaseCourtReport.new(
-        volunteer_id: current_user.id, # ??? not a volunteer ? linda
+        volunteer_id: current_user.volunteer? ? current_user.id : casa_case.assigned_volunteers.first.id,
```

Comment:
A volunteer is required to generate a report. So I'm always picking the first assigned volunteer. Is there another way you'd like to solve this problem?

Reply:
This seems reasonable for now because most cases have only one volunteer. Eventually we will want to be able to specify the volunteer, since some cases have multiple volunteers (think sibling youth, and a married pair of volunteers)- and I'm not sure whether both volunteers need to submit a court report for the same youth.